.circleci/config.yml

```diff
@@ -57,4 +59,8 @@ jobs:
 workflows:
   speech-processor-workflow:
     jobs:
-      - speech-processor
+      - hold:
+          type: approval
+      - speech-processor:
+          requires:
+            - hold
```

Comment:
Before enabling it automatically again: Check all the TODOs in the description of the PR + Clean commits